packages/sd-jwt/src/lib/crypto.ts

```diff
@@ -0,0 +1,16 @@
+// FIXME: create an hash function that works on nodejs, browser and react native runtimes
```

Comment:
really there is no library to do this?

Reply:
I haven't found a library that provides hashing on all three runtimes, yet. Neither I was able to find a react-native alternative for salted sha-256.

I opened a dedicated PR to propose a solution: https://github.com/pagopa/io-wallet/pull/7